application/forms/EventRuleConfigElements/EscalationCondition.php

```diff
@@ -129,28 +130,21 @@ protected function assemble(): void
                     break;
                 case 'incident_age':
                     $val = $this->createElement(
-                        'text',
+                        'number',
                         $valName,
                         [
                             'required'   => true,
-                            'class'      => ['autosubmit', 'right-operand'],
-                            'validators' => [
-                                new CallbackValidator(function ($value, $validator) {
-                                    if (! preg_match('~^\d+(?:\.?\d*)?[hms]{1}$~', $value)) {
-                                        $validator->addMessage(
-                                            $this->translate(
-                                                'Only numbers with optional fractions (separated by a dot)'
-                                                . ' and one of these suffixes are allowed: h, m, s'
-                                            )
-                                        );
-
-                                        return false;
-                                    }
-
-                                    $validator->clearMessages();
-                                    return true;
-                                })
-                            ]
+                            'class'      => 'right-operand',
+                            'value'      => 1
+                        ]
```

Comment:
Only whole numbers are supported now, `1.5h` is invalid.
- Please add `0.5` as step. 
- Remove second.
- Minimum allowed minute should be `1m`, not `0.5m`.